tests/fuzz/runFuzzTest.sh

```diff
@@ -177,7 +177,7 @@ then
         fi
 
         if [ "$FEATURE_COVERAGE" -lt $MIN_FEATURES_COVERED ]; then
-            printf "\033[33;1mWARNING!\033[0m ${TEST_NAME} only covers ${FEATURE_COVERAGE} features, which is below ${MIN_FEATURES_COVERED}! This is likely a bug.\n"
+            printf "\033[31;1mERROR!\033[0m ${TEST_NAME} only covers ${FEATURE_COVERAGE} features, which is below ${MIN_FEATURES_COVERED}! This is likely a bug.\n"
```

Comment:
Instead of a `likely a bug`, do we want to call out the possible missing corpus files ?

Reply:
I updated the message and also added a warning if the corpus directory does not exist